rt_gene/msg/MSG_Blink.msg

```diff
@@ -0,0 +1,2 @@
+string subject_id
+bool blink
```

Comment:
Can we add the estimate here as well? Might come in handy.

Reply:
I did think about it quite heavily - the initial idea was to just use the published probabilities but then it won't be using the threshold value that gets set in the node and thought it might be confusing.

My  thought was to create the blink threshold as a dynamically changing variable (like ModelSize), would that help?

Or I can just publish both, the boolean and the probability?

Reply:
Yes I was thinking both. The threshold can then be obtained from the parameter server if needed.

Reply:
I added a probability float to the message, I'll leave the dynamic parameter server stuff till later if that's okay?